### ISSUE TITLE
Add the export option in the "pubs list" command

### ISSUE THREAD

**Issue by amartos** [pubs/pubs#89]
For now, to `pubs export` the output of `pubs list` commands, a custom command needs to be set up. A simple option (as "-e") could be added, for simplicity. It would be useful to generate a bib file with tagged papers.

**Comment by omangin** [comment]
Yes, that's a good idea for improvement. The list command definitely needs more options for the print format. Maybe `bibtex` should be one of them. This however raises the question of how much the two commands overlap and whether we should keep both or merge them. Any though?

In the meanwhile you can pass several keys to the `export` command so a `search = !pubs list -k "$@" | xargs pubs export ` should do it too.

**Comment by amartos** [comment]
> This however raises the question of how much the two commands overlap and whether we should keep both or merge them. Any though?

From a purely academics/non-programmer-terminal-friendly personal POV, keeping the two commands separated is not a big deal. The main use of an export option in the `list` command would be for tagged papers, or for papers with a specific author, IMO. The export command do the rest just fine. I don’t see the difference between having one command with many options (and surely aliases), or two explicit/easy to remember commands.

Thanks for the alternate workaround (it is surely faster than mine).

**Comment by dotlambda** [comment]
It would suffice to let `pubs export` accept citation keys from stdin, so that one could use
```sh
pubs list --citekeys-only $query | pubs export -
```

**Comment by omangin** [comment]
@dotlambda, as mentioned above the `xargs` command can already read citation keys from `stdin` and use them as argument for pubs, as follows.
```bash
pubs list --citekeys-only $query | xargs pubs export
```

**Comment by zamlz** [comment]
I've just been doing,
```
pubs export `pubs list -k $query`
```
avoids the piping and the xargs
